src/pages/about/index.tsx

```diff
@@ -78,118 +78,116 @@ export default function About() {
+            </p>
+            <p>
+              Over the years, we have identified better ways of doing things and
+              novel use cases, and those insights have been rolled back into the
+              design of the software. When the authors of the software are also
+              some of its most demanding users, you get quality tools that solve
+              actual problems. We find these tools to be a joy to use, and we
+              think you will too.
+            </p>
           </Stack>
         </div>
-      </div>
 
-      <div className="my-3 py-5 bg-dark">
-        <div className="container-md">
-          <Row>
-            <Column>
-              <h2 className="text-white">Open-source. Forever.</h2>
-              <p className="text-white ">
-                PrairieLearn began life as open-source software, and we&apos;re
-                committed to making sure it stays that way. With an active
-                developer community of professors, course staff, and students,
-                PrairieLearn gets better all the time.
-              </p>
-              <div className="d-flex">
-                <a
-                  href="https://github.com/PrairieLearn/PrairieLearn"
-                  className="btn btn-light me-3"
-                >
-                  GitHub →
-                </a>
-                <a
-                  href="https://prairielearn.readthedocs.io/en/latest/"
-                  className="btn btn-light"
-                >
-                  Documentation →
-                </a>
-              </div>
-            </Column>
-          </Row>
+        <div className="container-md pt-2">
+          <Stack spacing={4}>
+            <Heading>PrairieLearn, Inc.</Heading>
+
+            <p>
+              PrairieLearn began life as open-source software, and it will
+              remain that way forever. This means that institutions and
+              individuals can run PrairieLearn on their own infrastructure and
+              make changes to it at will. PrairieLearn has a strong community of
+              developers who frequently contribute new features, improvements,
+              and fixes.
+            </p>
+
+            <p>
+              However, we&apos;ve found that most instructors would rather focus
+              on running their courses instead of operating and scaling
+              software. We started PrairieLearn, Inc. to provide a paid hosting
+              option. This ensures that everyone can take advantage of all that
+              PrairieLearn has to offer without having to become experts in
+              server administration.
+            </p>
+
+            <p>
+              Paid subscriptions cover the cost of running PrairieLearn, and
+              they also fund our developers that operate the service and are
+              responsible for ongoing maintenance, upgrades, code reviews, an
+              support.
+            </p>
```

Comment:
We can probably iterate on this text a bit.

Reply:
I added some suggested edits, but feel free to take what you want from that.